dask/dataframe/tests/test_dataframe.py

```diff
@@ -4307,6 +4334,51 @@ def test_dataframe_mode():
     assert_eq(ddf.mode(), df.mode(), check_index=False)
 
 
+def test_median():
+    df = pd.DataFrame({"x": [1, 2, 3, 4, 5], "y": [1.1, 2.2, 3.3, 4.4, 5.5]})
+    ddf = dd.from_pandas(df, npartitions=3)
+
+    # Exact medians work when `axis=1` or when there's only have a single partition
+    # TODO: shouldn't need to specify `check_names=False` below, but names currently don't match
+    assert_eq(ddf.median(axis=1), df.median(axis=1), check_names=False)
```

Comment:
I believe that we can remove the `check_names=False` by chaining a `.rename(None)` onto the dataframe median functions (the series ones are just scalars, so don't have names)

Reply:
Thanks for the suggestion @ian-r-rose. I went ahead and added `.rename(None)` and removed the `check_names=False` lines (and corresponding `TODO` comments). We don't often need to use `rename` so it still feels like there's something deeper happening with the name. That said, I don't want to block progress on this point -- adding a `rename` call is a valid thing to do

Reply:
I don't think there's really anything weird going on with the series name: it's just a difference between the behaviors of `quantile` and `median`. With pandas, each quantile series gets the name of the quantile, but median does not:

```python
In [1]: import pandas

In [2]: df = pandas.DataFrame({"a": [1,2,3,4], "b": [5,6,7,8]})

In [3]: df.median(axis=1)
Out[3]: 
0    3.0
1    4.0
2    5.0
3    6.0
dtype: float64

In [4]: df.quantile(axis=1)
Out[4]: 
0    3.0
1    4.0
2    5.0
3    6.0
Name: 0.5, dtype: float64
```

Reply:
Ah, great